docs/Tutorials/OpenAPI/Specification/v3.0.3.md

```diff
@@ -0,0 +1,4162 @@
+| <a name="serverVariableDescription"></a>description |  `string`  | An optional description for the server variable. [CommonMark syntax](https://spec.commonmark.org/) MAY be used for rich text representation.                                                                                                                                                                                                                                     |
+
+This object MAY be extended with [Specification Extensions](#specificationExtensions).
+
+#### <a name="componentsObject"></a>Components Object
+
+Holds a set of reusable objects for different aspects of the OAS.
+All objects defined within the components object will have no effect on the API unless they are explicitly referenced from properties outside the components object.
+
+
+##### Fixed Fields
+
+| Field Name                                               | Type                                                                                                   | Pode                                                                                                                    | Description                                                                  |
+| -------------------------------------------------------- | :----------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------- |
+| <a name="componentsSchemas"></a> schemas                 | Map[`string`, [Schema Object](#schemaObject) \| [Reference Object](#referenceObject)]                  | [`Add-PodeOAComponentSchema`](https://badgerati.github.io/Pode/Functions/OpenApi/Add-PodeOAComponentSchema)             | An object to hold reusable [Schema Objects](#schemaObject).                  |
+| <a name="componentsResponses"></a> responses             | Map[`string`, [Response Object](#responseObject) \| [Reference Object](#referenceObject)]              | [`Add-PodeOAComponentResponse`](https://badgerati.github.io/Pode/Functions/OpenApi/Add-PodeOAComponentResponse)         | An object to hold reusable [Response Objects](#responseObject).              |
+| <a name="componentsParameters"></a> parameters           | Map[`string`, [Parameter Object](#parameterObject) \| [Reference Object](#referenceObject)]            | [`Add-PodeOAComponentParameter`](https://badgerati.github.io/Pode/Functions/OpenApi/Add-PodeOAComponentParameter)       | An object to hold reusable [Parameter Objects](#parameterObject).            | PodeOAComponentExample |
```

Comment:
I would avoid hardcoding doc/function links for Pode's functions, and let Pode's doc-builder auto-map them during generation. This will mean links can be tested locally, and mapped correctly - there are a few that reference `/OpenApi/<func>` instead of `/OAComponent/<func>` for example 😋 

Similar to the other doc page, just wrap Pode functions that need links with [\`...\`]"/ Ie:

[\`Add-PodeOAComponentResponse\`]

rather than hardcoding

[\`Add-PodeOAComponentParameter\`]\(https://badgerati.github.io/Pode/Functions/OpenApi/Add-PodeOAComponentParameter)

Reply:
I was not aware of that. Now I understood why a lot of markdown documents are not working properly on github😀